### PR TITLE
Copy wrapper inputs to factory scratch directory

### PR DESCRIPTION
**Pull request by nkremerh**
Addresses #2106.

### CODE REVIEW

**Comment by nkremerh** [comment]
@btovar, thoughts?

File: batch_job/src/work_queue_factory.c [hunk 1]
```diff
@@ -1331,6 +1335,30 @@ int main(int argc, char *argv[])
 		return 1;
 	}
 
+	if(wrapper_input) {
+		struct jx *item;
+		for(void *i = NULL; (item = jx_iterate_array(wrapper_inputs, &i));) {
+			const char *value = item->u.string_value;
+			const char *file_at_scratch_dir = string_format("%s/%s", scratch_dir, path_basename(value));
+			int64_t result = 0;
```
Comment (dthain) [inline batch_job/src/work_queue_factory.c]: Save some space and use `copy_direntry` here, which handles both cases of file and dir...

Comment (nkremerh) [reply batch_job/src/work_queue_factory.c]: Done.

**Comment by dthain** [comment]
Ready?

**Comment by nkremerh** [comment]
Yes. Ready to merge.

**Pull request merged**